brand.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: fullwidth
+primary_title: Resources
+title: Brand Guidelines
+---
+
+## Brand Guidelines
+
+---
+
+We developed these guidelines to help keep the OpenSearch brand unified and cohesive. Please reference this guide for all communication and brand design across all formats.
+
+Download: <a href="/assets/media/brand/opensearch-brand-guide-2021.pdf" target="_blank">Brand Guidelines PDF</a>
+
+### Logos & Marks
+
+#### Logo
+<img src="assets/media/brand/SVG/Logo/opensearch_logo_default.svg" height="30px" alt="OpenSearch Logo" style="display: block; margin-bottom: 1em;" />
+Default: <a href="assets/media/brand/PNG/Logo/opensearch_logo_default.png" target="_blank">PNG</a> &middot; <a href="assets/media/brand/SVG/Logo/opensearch_logo_default.svg" target="_blank">SVG</a><br />
+Dark Mode: <a href="assets/media/brand/PNG/Logo/opensearch_logo_darkmode.png" target="_blank">PNG</a> &middot; <a href="assets/media/brand/SVG/Logo/opensearch_logo_darkmode.svg" target="_blank">SVG</a><br />
+Monochrome: <a href="assets/media/brand/PNG/Logo/opensearch_logo_monochrome.png" target="_blank">PNG</a> &middot; <a href="assets/media/brand/SVG/Logo/opensearch_logo_monochrome.svg" target="_blank">SVG</a>
+
+#### Mark
+<img src="assets/media/brand/SVG/Mark/opensearch_mark_default.svg" height="30px" alt="OpenSearch Mark" style="display: block; margin-bottom: 1em;" />
+Default: <a href="assets/media/brand/PNG/Mark/opensearch_mark_default.png" target="_blank">PNG</a> &middot; <a href="assets/media/brand/SVG/Mark/opensearch_mark_default.svg" target="_blank">SVG</a><br />
+Dark Mode: <a href="assets/media/brand/PNG/Mark/opensearch_mark_darkmode.png" target="_blank">PNG</a> &middot; <a href="assets/media/brand/SVG/Mark/opensearch_mark_darkmode.svg" target="_blank">SVG</a><br />
+Monochrome: <a href="assets/media/brand/PNG/Mark/opensearch_mark_monochrome.png" target="_blank">PNG</a> &middot; <a href="assets/media/brand/SVG/Mark/opensearch_mark_monochrome.svg" target="_blank">SVG</a>
+
+### More file formats
+These contain the full set of PNG, SVG and EPS Logos, Marks & Logotypes as downloadable .zip files.
+
+Download: <a href="assets/media/brand/opensearch-assets_SVG.zip" target="_blank">SVG</a> &middot; <a href="assets/media/brand/opensearch-assets_PNG.zip" target="_blank">PNG</a> &middot; <a href="assets/media/brand/opensearch-assets_EPS.zip" target="_blank">EPS</a>
+
+<br />
```

Comment:
Do we need this br ;)

Reply:
temporary hack to add a bit of additional space between the content and the footer. I'll address it properly via CSS later.